src/get-glamor-classname.js

```diff
@@ -33,18 +39,14 @@ function getGlamorClassName({
   theme,
   context,
 }) {
-  const {
-    glamorStyles: parentGlamorStyles,
-    glamorlessClassName,
-  } = extractGlamorStyles(props.className)
   const {mappedArgs, nonGlamorClassNames} = handleStyles(
-    [...styles, parentGlamorStyles, cssOverrides, cssProp],
+    [...styles, props.className, cssOverrides, cssProp],
```

Comment:
Actually, now that I think of it, this change probably would have worked before as well 😅